### DISCUSSION TITLE
Chines 10.2 inventer

### DISCUSSION THREAD

**Discussion by Ensferi** [odya/esphome-powmr-hybrid-inverter#40] [Q&A]
Hello,
 please i have this inventer 
https://www.aliexpress.com/item/1005006519140238.html?src=google&src=google&albch=shopping&acnt=494-037-6276&slnk=&plac=&mtctp=&albbt=Google_7_shopping&albagn=888888&isSmbAutoCall=false&needSmbHouyi=false&albcp=19367082846&albag=&trgt=&crea=en1005006519140238&netw=x&device=c&albpg=&albpd=en1005006519140238&gad_source=1&gclid=EAIaIQobChMI36jfoePNhQMVAa5oCR2bpQzLEAQYAiABEgKrS_D_BwE&gclsrc=aw.ds&aff_fcid=f2a5e0bdf82e46069f5dddd7236a0de9-1713513196612-05946-UneMJZVf&aff_fsk=UneMJZVf&aff_platform=aaf&sk=UneMJZVf&aff_trace_key=f2a5e0bdf82e46069f5dddd7236a0de9-1713513196612-05946-UneMJZVf&terminal_id=10b45cf2800d411fa13224c2e342fced&afSmartRedirect=y

and its look same as powmr or its o copy of powmr ?
 so please will this HA integration work with my inventer ? or should i use pipsolar ?
thanks you

**Comment by odya** [comment]
I don't think it's similar to PowMr. Different screen, additional ports.

**Comment by Ensferi** [comment]
yes different screen but the old one have the same screen and same rs port, what i have now is new version but it should be the same.
 now i need help with install this in esp32

**Comment by Patogoma** [comment]
Install syssi pipsolar version 8048. I have an Anern 10.2 and is the same inverter. All works correctly except PV2 data.

**Reply by paivatranslda** [reply]
Because the isn't any PV2, it is a single MPPT, dual input.  Thanks for the hints!

**Reply by patogomaII** [reply]
No. It have a dual PV input. Actually I have running the correct modbus code reading both PV inputs via serial connection with a simple hack of the esp8266 based wifi dongle. ;-P
